src/Uno.UI.RuntimeTests/Tests/NativeCtorGeneratorTests/GameBoardCanvas.cs

```diff
@@ -0,0 +1,20 @@
+#if __ANDROID__
+using Android.Content;
+
+namespace Uno.UI.RuntimeTests.Tests.NativeCtorGeneratorTests
+{
+	public partial class GameBoardCanvas : SkiaSharp.Views.Android.SKGLTextureView
```

Comment:
Let's take this a different way. Why do you need this specific type ? `SKGLTextureView` ? Can't you use one that comes with the android API?